src/main/java/org/prebid/server/bidder/openx/OpenxBidder.java

```diff
@@ -173,6 +173,16 @@ private Imp makeImp(Imp imp) {
         return impBuilder.build();
     }
 
+    private BigDecimal resolveBidFloor(Imp imp, ExtImpOpenx openxExtImp) {
+        BigDecimal impBidFloor = imp.getBidfloor();
+        BigDecimal customFloor = openxExtImp.getCustomFloor();
+        if (impBidFloor == null || impBidFloor.compareTo(BigDecimal.ZERO) == 0
```

Comment:
In Go they have 0 as default number value, for better logic lets check that is more than 0(handle case with negative numbers